src/openrct2/rct12/RCT12.cpp

```diff
@@ -477,7 +478,7 @@ bool RCT12PathElement::IsBlockedByVehicle() const
     return (flags & RCT12_TILE_ELEMENT_FLAG_BLOCKED_BY_VEHICLE) != 0;
 }
 
-bool RCT12TrackElement::BlockBrakeClosed() const
+bool RCT12TrackElement::GetBrakeClosed() const
```

Comment:
This should not have been changed. Its still the block brake closed.

Reply:
Regular brakes have a closed state in this PR.

Reply:
I missed the RCT12 part. This is now fixed.